### ISSUE TITLE
Breakpoints are not hit when debugging a unit test in VSCode

### ISSUE THREAD

**Issue by spektom** [rust-analyzer/rust-analyzer#9324]
Breakpoints are set, but debugging a unit test doesn't stop on them.
No errors/warnings are shown.

__VSCode__

Version: 1.58.0-insider (user setup)
Commit: a81fff00c9dab105800118fcf8b044cd84620419
Date: 2021-06-17T05:17:34.858Z
Electron: 12.0.11
Chrome: 89.0.4389.128
Node.js: 14.16.0
V8: 8.9.255.25-electron.0
OS: Windows_NT x64 10.0.19043

__VSCode Extensions__

Rust Analyzer: v0.2.637
C/C++ (ms-vscode.cpptools): v1.5.0-insiders

**Comment by lnicola** [comment]
`rust-analyzer` itself doesn't do much in the way of debugging. It might be better to file this against the debugging extension.

In my previous tests, some extensions worked better than others, depending on your platform.

**Comment by spektom** [comment]
Got it, thanks.

**Comment by lnicola** [comment]
Is debugging working for anyone?

**Comment by momvart** [comment]
For those who are not getting any of their breakpoints hit: In my case, it was a problem with the path of project files.
I had opened my project in vscode using a symbolically linked directory and vscode was using the virtual path while LLDB was looking at files with real paths. So, LLDB was not recognizing any of those breakpoints that I had set.

Solution: Open the folder with the real path in vscode.
